torch/utils/_cxx_pytree.py

```diff
@@ -42,19 +42,13 @@
 
 
 __all__ = [
-    "PyTree",
-    "Context",
-    "FlattenFunc",
-    "UnflattenFunc",
-    "DumpableContext",
-    "ToDumpableContextFunc",
-    "FromDumpableContextFunc",
```

Comment:
These type aliases are removed from `__all__`. Otherwise, the public API test will ask to add these to `test/allowlist_for_publicAPI.json`.